.dockerignore

```diff
@@ -2,6 +2,6 @@
 *Dockerfile*
 *docker-compose*
 */node_modules
-bin
```

Comment:
Please keep these. They're from python virtual environments.

Reply:
I updated the docs to recommend `python3 -m venv venv` instead of `python3 -m venv .`, but I suppose it's not too much trouble to support both use cases.

Reply:
Fixed in 17291c9.

Reply:
I'm curious, why the change?

Reply:
Personally, having all virtual environment-related support files in a single folder instead of being scattered throughout the top level directory feels neater to me. But I realize that's a personal preference so 🤷

Reply:
Fair point, personally I'm just lazy and don't like typing an extra directory name when I activate the environment.